src/frontend/src/lib/components/ui/Logo.svelte

```diff
@@ -36,6 +37,7 @@
 	class:bg-off-white={color === 'off-white' && !loaded}
 	class:bg-white={color === 'white' && !loaded}
 	class:opacity-10={!loaded}
+	class:invisible
```

Comment:
Opiniated answer but, I'm not a big fan of having elements in the DOM that are always invisible for aligment. Can we not render the element and use CSS to align the elements?

Reply:
eheh I agree with you actually. This was the quick solution, not the optimal, let me check

Reply:
can it be something like this? or you meant rendering nothing at all and use just CSS to align?

```svelte
{#if invisible}
	<div style={`width: ${...}; height: ${...};`}></div>
{:else}
	<Logo ... />
{/if}
```

Reply:
otherwise:

```svelte
<div class="flex-none w-14" slot="icon">
	{#if !invisibleLogo}
		<Logo
			color="white"
			src={token.network.icon}
			alt={replacePlaceholders($i18n.core.alt.logo, { $name: token.network.name })}
			size="medium"
		/>
	{/if}
</div>
```

Reply:
I would use a CSS grid and set start to 1 or 2 depending if there is a logo or none to render.

Reply:
Check last commit please

Reply:
Thx